README.md

```diff
@@ -24,58 +24,67 @@ pip3 install --upgrade datajoint
 ```
 ## Python Native Blobs
 
-For the v0.12 release, the variable `enable_python_native_blobs` can be
-safely enabled for improved blob support of python datatypes if the following
-are true:
+DataJoint 0.12 adds full support for all native python data types in blobs: tuples, lists, sets, dicts, strings, bytes, `None`, and all their recursive combinations.
+The new blobs are a superset of the old functionality and are fully backward compatible.
+In previous versions, only MATLAB-style numerical arrays were fully supported.
+Some Python datatypes such as dicts were coerced into numpy recarrays and then fetched as such.
 
-  * This is a new DataJoint installation / pipeline(s)
-  * You have not used DataJoint prior to v0.12 with your pipeline(s)
-  * You do not share blob data between Python and Matlab
+However, since some Python types were coerced into MATLAB types, old blobs and new blobs may now be fetched as different types of objects even if they were inserted the same way. 
+For example, new `dict` objects will be returned as `dict` while the same types of objects inserted with `datajoint 0.11` will be recarrays.
 
-Otherwise, please read the following carefully:
+Since this is a big change, we chose to disable full blob support by default as a temporary precaution, which will be removed in version 0.13.
+
+You can enable it by setting the `enable_python_native_blobs` flag in `dj.config`. 
+
+```python
+import datajoint as dj
+dj.config["enable_python_native_blobs"] = True
+```
+
+You can safely enable this setting if both of the following are true:
```

Comment:
I like the new list of improvements, etc at the intro -

However, since this is a migration note, we did discuss having some sort of 'quick skim list' at the top of the section for the people who don't want to read anything and just use the new feature, and to make hitting this error less intimidating for new users.

The new initial text is presented more positively, which I think is good, but pushes down the 'please set this setting if XYZ doesnt apply' block from the beginning.

Reply:
Yes, I think it's important to explained what happened with a more accurate and complete, not only more positive explanation of what happened. The original explanation started mid-sentence without providing a context.